x-pack/plugins/cases/common/api/cases/user_actions/response.ts

```diff
@@ -85,9 +82,7 @@ const CaseUserActionResponseWithoutActionIdRt = rt.intersection([
 
 export const CaseUserActionAttributesRt = CaseUserActionBasicRt;
 export const CaseUserActionsResponseRt = rt.array(CaseUserActionResponseRt);
-export const CaseUserActionsResponseWithoutActionIdRt = rt.array(
-  CaseUserActionResponseWithoutActionIdRt
-);
+export const CaseUserActionsResponseWithoutActionIdRt = rt.array(CaseUserActionResponseRt);
```

Comment:
I think this should be `CaseUserActionsResponseRt` instead of  `CaseUserActionsResponseWithoutActionIdRt`